content/developer/misc/other/cmdline.rst

```diff
@@ -259,6 +259,10 @@ Emails
 
     Email address used as <FROM> when Odoo needs to send mails
 
+.. option:: --from-filter <address or domain>
+
+    Email address or domain authorized to send outgoing mails
```

Comment:
It's not exactly that, it's more
> Email address or domain that the mail server is configured for. If the email of the sender do not match this value, it might be encapsulated into `mail.default.from`

(if the email address do not match this, and if you have only 1 mail server, the email will be sent anyway, so the word "authorized" is not 100% correct)